### DISCUSSION TITLE
Aurora seems to go to sleep

### DISCUSSION THREAD

**Discussion by ChrisL79** [magicbug/Cloudlog#2815] [Aurora]
If i log a number of stations in succession, things work fine. If there is a delay of a few minutes before logging the next qso, Aurora will not send any data to Cloudlog.

I have tried running the program as administrator and regular user.

Using Windows 7 x64.

There is nothing wrong with the setup as Gridtracker uploads to Cloudlog just fine.

I think there is either a bug in Aurora or it does not play nicely with W7x64

Any ideas would be appreciated as this is the final piece in getting everything up and running.

Thanks,

Chris.

**Comment by pd5wl** [comment]
I think i have the same experience.
Latest Aurora
JTDX v2.2.159
Win 10 Pro,

Recently cwitcht from adifpush and wsjt-z  to this setup.

First QSO nothing happens, 2nd got logged, while changing the looks and feel of JTDX after a few minustes made an other QSO.. nothing happend, no data in CloudLog.

Restart of Aurora, made another QSO. again nothing happens. 

Also Lookup doesnt work from JTDX.

Is there anywhere I need to look or find logs to see what happens.

**Reply by ChrisL79** [reply]
I gave up on it in the end.

I worked with the author of Log4om to implement auto upload to cloudlog, so there is no need for any program like this now (as long as you use Log4om of course).

**Comment by pd5wl** [comment]
Using WSJT-X with Aurora works fine.

I just wonder, in JTDX i have setup up TCP as well as UDP. Might that be the trick? 
Aurora is working just not so good with JTDX.

**Reply by ChrisL79** [reply]
I tried all ways. Was still like it went to sleep after the first qso logged no matter what I tried.

**Reply by pd5wl** [reply]
So far WSJT-X/Z dont have a problem. Even not after a few minutes setting up a cron on my vps.. took me 4 mins or so to make it work. 
So i am currious how to debug this. 

@magicbug any options to debug this behaiviour?
